smf/mgs-sim/config.toml

```diff
@@ -16,7 +16,7 @@ request_body_max_bytes = 536870912
 [switch]
```

Comment:
These SMF config changes don't seem related to thing-flinger -- are they?

Reply:
Oh jeez. This is what I get for pushing at 2 am. I must have done a `git add .` with the mods from the testbed. Thanks Sean!